sdk/README.md

```diff
@@ -80,36 +80,44 @@ node index.js
 
 ### Configure your blockchain parameters
 
-You can also define your blockchain application parameters such as `BLOCK_TIME`, `MAX_TRANSACTIONS_PER_BLOCK` and more with an optional configurations object.
+You can also define your blockchain application parameters such as `blockTime`, `maxPayloadLength` and more with an optional configurations object.
 
 ```js
-const app = new Application(genesisBlockDevnet, {
-    app: {
-        label: 'my-blockchain-application',
-        genesisConfig: {
-            BLOCK_TIME: 10,
-            MAX_TRANSACTIONS_PER_BLOCK: 25,
-        },
-        ...
+const app = Application.defaultApplication(genesisBlockDevnet, {
+    genesisConfig: {
+        communityIdentifier: 'newChain',
+        blockTime: 5,
+        maxPayloadLength: 100 * 1024,
+        minRemainingBalance: "5000000",
+		activeDelegates: 101,
+		standbyDelegates: 2,
+		delegateListRoundOffset: 2
+    },
+    ...
 });
 ```
 
 For a complete list of configuration options see the [lisk-docs repo](https://github.com/LiskHQ/lisk-docs/blob/development/lisk-sdk/configuration.md).
 
-### Register a custom transaction
+### Register a custom module or a custom plugin
 
-You can [define your own transaction types](https://github.com/LiskHQ/lisk-docs/blob/master/lisk-sdk/customize.md) with Lisk-SDK. This is where the custom logic for your blockchain application lives.
+A custom module is a logic to define state changes which will be executed on-chain meaning that it will be a part of blockchain protocol.
+On the other hand, custom plugin is a logic to define a off-chain logic which is not part of the blockchain protocol but to enhance the application features.
```

Comment:
```suggestion
A custom module is a logic to define state changes that will be executed on-chain meaning that it will be a part of the blockchain protocol.
On the other hand, a custom plugin is a logic to define an off-chain logic which is not part of the blockchain protocol but to enhance the application features.
```